src/lib.rs

```diff
@@ -180,8 +180,30 @@ mod contract {
     /// code.
     #[no_mangle]
     pub extern "C" fn state_migration() {
-        // This function is purposely left empty because we do not have any state migration
-        // to do.
+        // Only owner can call migration
+        sdk::assert_private_call();
```

Comment:
Since @artob said he does not use the state migration mechanism, I think we should remove the implementation here, change `Connector::get_metadata` to return an `Option` (may or may not be present in storage), and then in `ft_metadata()` provide our default value (symbol = ETH, name = Ether, etc) in the `None` case.